include/git2/diff.h

```diff
@@ -130,7 +130,9 @@ typedef enum {
 	GIT_DIFF_INCLUDE_CASECHANGE = (1u << 11),
 
 	/** If the pathspec is set in the diff options, this flags means to
-	 *  apply it as an exact match instead of as an fnmatch pattern.
```

Comment:
Nah, if there's confusion then I failed at writing the docs.  How about:

>  If the pathspec is set in the diff options, this flag indicates that the list will be treated as literal paths instead of fnmatch patterns.  Each path in the list must either be a full path to a file or directory (a trailing slash indicates that the path will *only* match a directory).  If a directory is specified, all children will be included.